CUMULUS-2068 Dropdown update

dopeters opened the pull request:
**Summary:** Summary of changes

Addresses [CUMULUS-2068: Dashboard: UI Interaction Fix - Dropdowns](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2068)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

jjmccoy commented:
Nice work. The interaction and dropdown look great! Thank you. I noticed some hover and selection wonkiness when it comes to styling. 

After selection and mouse moved away it still shows the hover state:
<img width="1058" alt="Screen Shot 2020-08-18 at 7 07 45 PM" src="https://user-images.githubusercontent.com/40603352/90574295-8e732700-e186-11ea-9584-c4df065f41b9.png">

For dropdown item hover: the blue browser default is overriding our custom link blue:
<img width="876" alt="Screen Shot 2020-08-18 at 7 00 31 PM" src="https://user-images.githubusercontent.com/40603352/90574396-ce3a0e80-e186-11ea-836c-d6743c9169d3.png">

dopeters commented:
@jjmccoy I added the prop `blurInputOnSelect={true}` to the Select component to unfocus the input after selecting the option.

I also updated the background color of hovering on the option. I also just made some updates so that some of the styles in SimpleDropdown are being pulled as variables from our css files so that future changes will be easier. Let me know what you think!

jjmccoy commented:
> @jjmccoy I added the prop `blurInputOnSelect={true}` to the Select component to unfocus the input after selecting the option.
> 
> I also updated the background color of hovering on the option. I also just made some updates so that some of the styles in SimpleDropdown are being pulled as variables from our css files so that future changes will be easier. Let me know what you think!

@dopeters Oh cool! Yeah trying to use CSS variables more. They make it so much better. Thanks.

I went back looked at the mockups and noticed the styling. My bad for not including this in the ticket as a reference for you.  Would it be too much trouble to make the inline hover #EBF3F8 for the background and we will keep the selected background color our `$ocean-blue`? Let me know if it's a bother. Also, I noticed that the dropdown option arrows are supposed to be white in all states -- it's a little hard to see the grey on blue contrast. 

![Cumulus Dashboard - Home - Filters Shown](https://user-images.githubusercontent.com/40603352/90831101-fce3f080-e310-11ea-9334-f813a720f290.png)
